src/app/zap-templates/zcl/data-model/chip/application-basic-cluster.xml

```diff
@@ -26,12 +26,15 @@ limitations under the License.
     <description>This cluster provides information about an application running on a TV or media player device which is represented as an endpoint.</description>
     <attribute side="server" code="0x0000" define="APPLICATION_VENDOR_NAME"         type="char_string"                default=""     length="32"               writable="false" optional="true">VendorName</attribute>
     <attribute side="server" code="0x0001" define="APPLICATION_VENDOR_ID"           type="vendor_id"                  default="0x0"  min="0x0000" max="0xFFFF" writable="false" optional="true">VendorID</attribute>
-    <attribute side="server" code="0x0002" define="APPLICATION_NAME"                type="char_string"                               length="32"               writable="false" optional="false">ApplicationName</attribute>
+    <attribute side="server" code="0x0002" define="APPLICATION_NAME"                type="long_char_string"           length="256"               writable="false" optional="false">ApplicationName</attribute>
```

Comment:
Hmm.  Is there a spec issue tracking the fact that this has a "desc" constraint when it should hust have a normal constraint?

Reply:
Not that I am aware ... only saw this because I saw the scraper saying "no constraint" and reading the description said 256 .... it should actually be 256 I guess but from an SDK perspective I would prefer bytes.